project2/styles.css

```diff
@@ -0,0 +1,201 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    color: #686868;
+    font-size: 18px;
+}
+
+section {
+    background-color: #faf4e3;
+    max-width: 640px;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+header {
+    background-image: url(images/shrimps_2.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position-y: center;
+    background-position-x: center;
+    height: 300px;
+    width: 640px;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
The header is a block element, so you can delete these 3 lines and see no change 👍